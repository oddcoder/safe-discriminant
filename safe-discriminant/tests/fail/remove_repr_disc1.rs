use safe_discriminant::Discriminant;
use safe_discriminant_derive::remove_repr;

#[remove_repr]
#[derive(Discriminant)]
#[repr(u8)]
pub enum Foo {
    A = 0,
    B = 1,
}

fn main() {}
