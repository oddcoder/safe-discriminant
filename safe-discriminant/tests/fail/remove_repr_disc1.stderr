error: Discriminant requires a `#[repr(x)] where x is one of u8, i8, u16, i16, u32, i32, u64, i64, u128, i128.
 --> tests/fail/remove_repr_disc1.rs:7:10
  |
7 | pub enum Foo {
  |          ^^^
